docs/usage.md

```diff
@@ -18,37 +14,40 @@ You will need to create a samplesheet with information about the samples you wou
 
 ### Multiple runs of the same sample
 
-The `sample` identifiers have to be the same when you have re-sequenced the same sample more than once e.g. to increase sequencing depth. The pipeline will concatenate the raw reads before performing any downstream analysis. Below is an example for the same sample sequenced across 3 lanes:
+The `sample` identifiers have to be the same when you have re-sequenced the same sample more than once e.g. to increase sequencing depth. The pipeline will concatenate the raw reads before performing any downstream analysis. Below is an example for the same sample sequenced across replicates 1 and 4 (`REP1` and `REP4` respectively):
 
 ```csv title="samplesheet.csv"
-sample,fastq_1,fastq_2
-CONTROL_REP1,AEG588A1_S1_L002_R1_001.fastq.gz,AEG588A1_S1_L002_R2_001.fastq.gz
-CONTROL_REP1,AEG588A1_S1_L003_R1_001.fastq.gz,AEG588A1_S1_L003_R2_001.fastq.gz
-CONTROL_REP1,AEG588A1_S1_L004_R1_001.fastq.gz,AEG588A1_S1_L004_R2_001.fastq.gz
+sample,fastq,cell_count
+CONTROL_REP1,AEG588A1_S1.fastq.gz,5000
+CONTROL_REP1,AEG588A1_S2.fastq.gz,5000
+CONTROL_REP2,AEG588A2_S1.fastq.gz,5000
+CONTROL_REP3,AEG588A3_S1.fastq.gz,5000
+CONTROL_REP4,AEG588A4_S1.fastq.gz,5000
+CONTROL_REP4,AEG588A4_S2.fastq.gz,5000
+CONTROL_REP4,AEG588A4_S3.fastq.gz,5000
 ```
 
-### Full samplesheet
+`cell_count` refers to the expected number of cells for each sample, which can vary from sample to sample.
 
-The pipeline will auto-detect whether a sample is single- or paired-end using the information provided in the samplesheet. The samplesheet can have as many columns as you desire, however, there is a strict requirement for the first 3 columns to match those defined in the table below.
+### Full samplesheet
```

Comment:
I would flip the order of the "Full samplesheet" section and the "multiple runs" section

Reply:
Good idea! Done with commit 891b67d